packages/forms/src/components/Repeater/Repeater.tsx

```diff
@@ -145,98 +184,158 @@ const Fieldset = ({
-            {children}
-          </div>
-          <div
-            data-h2-flex-shrink="base(0)"
+            data-h2-padding="base(x.5, x1, x1, x1)"
             data-h2-display="base(flex)"
             data-h2-flex-direction="base(column)"
-            data-h2-gap="base(x.25, 0)"
-            data-h2-order="base(1)"
+            data-h2-align-items="base(flex-start)"
+            data-h2-gap="base(x.5)"
+            data-h2-background-color="base(foreground)"
           >
             <div
               data-h2-background-color="base(foreground)"
               data-h2-display="base(flex)"
-              data-h2-radius="base(s)"
-              data-h2-shadow="base(medium)"
-              data-h2-flex-direction="base(column)"
+              data-h2-justify-content="base(space-between)"
               data-h2-align-items="base(center)"
-              data-h2-overflow="base(hidden)"
+              data-h2-width="base(100%)"
             >
-              <ActionButton
-                disabled={disabled || index <= 0}
-                onClick={decrement}
-                decrement
-                animate={!shouldReduceMotion}
-                aria-label={intl.formatMessage(formMessages.repeaterMove, {
-                  from: position,
-                  to: position - 1,
-                })}
+              <div
+                data-h2-display="base(flex)"
+                data-h2-align-items="base(center)"
+                data-h2-margin-left="base(-x.5)"
+                data-h2-justify-content="base(center)"
+                data-h2-gap="base(x.5)"
               >
-                <ChevronUpIcon data-h2-width="base(x1)" />
-              </ActionButton>
-              {!hideIndex && (
-                <span
-                  aria-hidden="true"
-                  data-h2-text-align="base(center)"
-                  data-h2-font-weight="base(700)"
-                  data-h2-margin="base(x.25, 0)"
-                >
-                  {index + 1}
-                </span>
-              )}
-              <ActionButton
-                disabled={disabled || index === total - 1}
-                onClick={increment}
-                animate={!shouldReduceMotion}
-                aria-label={intl.formatMessage(formMessages.repeaterMove, {
-                  from: position,
-                  to: position + 1,
-                })}
+                {!isLocked ? (
+                  <>
+                    <ActionButton
+                      disabled={isFirstItem || previousItemLocked}
+                      onClick={decrement}
+                      decrement
+                      animate={!shouldReduceMotion}
+                      aria-label={intl.formatMessage(
+                        formMessages.repeaterMove,
+                        {
+                          from: position,
+                          to: position - 1,
+                        },
+                      )}
+                      data-h2-margin="base(0, x.25, 0, 0)"
```

Comment:
Use of `gap` makes this unnecessary. I think we can remove most of the margin attributes now.

Reply:
Fixed by [83cffcb](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8181/commits/83cffcb65189d8cfbe5e36e86e62495e1e28afc2)